docs/docs/spec/metrics/metrics-examples.md

```diff
@@ -16,13 +16,13 @@ Some metrics and endpoints are very straightforward, but having a reference for
 ## Prometheus Metrics
 
 ### Economics metrics
-* `eigen_fees_earned_total{token="ETH"}`
-* `eigen_slashing_incurred_total{token="ETH"}`
-* `eigen_total_balance_total{token="ETH"}`
+* `eigen_fees_earned_total{token="ETH", unit="wei", strategy="eigenDA"}`
```

Comment:
remove eigenDA here, we haven't announced or even mentioned a token ever, and it's not clear we'll even have one.
Use 0x1BeE69b7dFFfA4E2d53C2a2Df135C388AD25dCD2 for the strategy, which is the address of the reth strategy (https://github.com/Layr-Labs/eigenlayer-contracts/blob/5d7b28e5af7db82918d22980030b926a055b8d79/script/output/M1_deployment_mainnet_2023_6_9.json#L20C16-L20C58)

Reply:
Can we also add a note that fees are yet to be implemented, so this metric should be optional, and obviously doesn't need to be implemented at this point in time.

Reply:
Yes, we can add such a note. Should the token here be `rETH`?

Reply:
Sure. Eventually the token label won't make sense, since strategies will be more generic and could contain multiple tokens. At that point I feel like the mapping from strategy->description (token list for eg) should be on grafana directly. Although maybe this is bad practice for operators who don't use grafana... wdyt here? I'm happy to keep add both strategy and token for now, and revisit this choice later when strategies become more than just LSTs.

Reply:
I see it will be helpful to keep both strategy and token for now. Let's revisit this later.

Reply:
Done [d0e911](https://github.com/NethermindEth/el-node-spec/pull/58/commits/3d0e911b83f03ccb7740f16dddcf4ed43ee53e72)